### PR TITLE
2nd task has been done

### PR DESCRIPTION
**Pull request by VolhaZh**

### CODE REVIEW

**Review by Alex99905 — APPROVED** [review]

**Review by breackinumber — CHANGES_REQUESTED** [review]
Good, but with some problems.

**Review by olgaz-skiba — APPROVED** [review]

File: homework/english/Volha_Zharava/lecture1/task2.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+You dont need to boot Sonar program
+I have to create a new request on GitHub
+My account connect to the main repository
```
Comment (breackinumber) [inline homework/english/Volha_Zharava/lecture1/task2.md:3]: **is connected**

File: homework/english/Volha_Zharava/lecture1/task2.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Cannot remove objects while the designer is in debug mode
+Please note that you cannot delete a post once placed on Github
+I want develop a chatbot while learning this course
```
Comment (breackinumber) [inline homework/english/Volha_Zharava/lecture1/task2.md:6]: want **to** develop

File: homework/english/Volha_Zharava/lecture1/task2.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+I cant disable this plugin
+I disconnect this zoom session now
```
Comment (breackinumber) [inline homework/english/Volha_Zharava/lecture1/task2.md:8]: I've been dissconnected from this zoom session

File: homework/english/Volha_Zharava/lecture1/task2.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+Result of running programm display at a terminal window
```
Comment (breackinumber) [inline homework/english/Volha_Zharava/lecture1/task2.md:9]: program**m** - program

File: homework/english/Volha_Zharava/lecture1/task2.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+Sonar verify programm code
```
Comment (breackinumber) [inline homework/english/Volha_Zharava/lecture1/task2.md:10]: program**m** - program

**Review by breackinumber — CHANGES_REQUESTED** [review]
Comments were added

File: homework/english/Volha_Zharava/lecture1/task2.md (hunk 4 above)
Comment (VolhaZh) [reply homework/english/Volha_Zharava/lecture1/task2.md:9]: agree
thnx

File: homework/english/Volha_Zharava/lecture1/task2.md (hunk 5 above)
Comment (VolhaZh) [reply homework/english/Volha_Zharava/lecture1/task2.md:10]: ok

**Review by Alex99905 — APPROVED** [review]

**Review by breackinumber — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]

**Review by breackinumber — COMMENTED** [review]
All fixed

**Review by Alex99905 — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]

**Pull request merged**